src/main/scala/com/github/unchama/seichiassist/subsystems/gachapoint/domain/BatchUsageSemaphore.scala

```diff
@@ -27,6 +27,12 @@ class BatchUsageSemaphore[F[_]: FlatMap, G[_]: ContextCoercion[*[_], F]](
       }.flatTap(grantAction.give)
     }(BatchUsageSemaphore.usageInterval)
 
+  def tryOneStackBatchTransaction: F[Unit] =
```

Comment:
github の仕様で suggestion が書けなかったのですが、`tryBatchTransaction` 関数は `tryLargeBatchTransaction` とするとよいと思います。

```suggestion
  def trySmallBatchTransaction: F[Unit] =
```